conf/my_custom.conf.dist

```diff
@@ -14,4 +14,4 @@
 #                     1 - Enabled
 #
 
-MyModule.Enable = 1
+MyModule.Enable = 0
```

Comment:
I would put it as 1 by default, because the developer that will use this, will surely set it as 1 to work on it